go.mod

```diff
@@ -66,7 +66,7 @@ require (
 )
 
 require (
-	github.com/consensys/gnark-crypto v0.9.2
+	github.com/consensys/gnark-crypto v0.9.1
```

Comment:
Why do we want to switch to `0.9.1`?

Reply:
the circuits were originally compiled with this version, so I just wanted to be safe. gnark emits warnings when the patch versions are not the same.